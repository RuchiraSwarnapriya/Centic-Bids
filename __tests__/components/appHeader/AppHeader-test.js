import React from 'react';
import renderer from 'react-test-renderer';
import AppHeader from '../../../src/components/appHeader/Header';


test('renders correctly', () => {
  const tree = renderer.create(<AppHeader />).toJSON();
  expect(tree).toMatchSnapshot();
});