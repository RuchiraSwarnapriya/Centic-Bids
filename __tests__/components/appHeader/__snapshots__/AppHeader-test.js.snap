// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flexDirection": "row",
      "justifyContent": "space-between",
      "marginLeft": 15,
      "marginRight": 30,
      "marginTop": 15,
    }
  }
>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <Text
      allowFontScaling={false}
      style={
        Array [
          Object {
            "color": "#000000",
            "fontSize": 45,
          },
          undefined,
          Object {
            "fontFamily": "Ionicons",
            "fontStyle": "normal",
            "fontWeight": "normal",
          },
          Object {},
        ]
      }
    >
      ïˆˆ
    </Text>
  </View>
  <Text
    style={
      Object {
        "fontSize": 25,
        "fontWeight": "bold",
      }
    }
  />
</View>
`;
